lambda-buffers-frontend/resources/good/Test.lbf

```diff
@@ -0,0 +1,30 @@
+module Test
```

Comment:
Will the module export all the definitions always?

Reply:
Yes exactly, I didn't give too much thought, but I struggle to find a reason to hide exports. The competition does the same really.

I imagine what would it mean for a type to be 'module private'? If it's used as a constituent in any other exported type, then it has to be exported as well, at least for codegen and user api (you need to construct the exposed type). If it's not used anywhere, then what's the point.

We can definitely visit this topic and give it a proper discussion.